link/src/main/java/com/stripe/android/link/account/AlreadyLoggedInLinkException.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android.link.account
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.link.model.AccountStatus
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class AlreadyLoggedInLinkException(
```

Comment:
Internally in `LinkAccountManager` no but if someone decides to log it externally it would result in an `unknown`, I'll change it to be a `StripeException`.